csdmpy/__init__.py

```diff
@@ -280,3 +280,21 @@ def plot(csdm_object, reverse_axis=None, range=None, **kwargs):
         >>> cp.plot(data_object) # doctest: +SKIP
     """
     return _preview(csdm_object, reverse_axis, range, **kwargs)
+
+
+def join(objects):
+    """Join dependent-variables of CSDM objects sharing the same dimensions
+    into one CSDM object.
+
+    Args:
+        objects: A list of CSDM objects with same dimensions.
+    """
+    new_obj = objects[0].copy()
+    for item in objects[1:]:
+        if item.dimensions == new_obj.dimensions:
```

Comment:
We may want this to be within machine precision instead of == since we're making that change soon

Reply:
That will be a part of a different PR. The function call for `==` is `def __eq__()`. The machine precision check will go inside this function. For user level, it will still appear as `==`